src/aux-server/aux-web/aux-auth/site/AuthApp/AuthApp.ts

```diff
@@ -38,6 +45,18 @@ export default class AuthApp extends Vue {
     comId: string = null;
     usePrivoLogin: boolean = false;
 
+    get studioNameFieldClass() {
+        return this.showEnterStudioNameError || this.showCreateStudioError
+            ? 'md-invalid'
+            : '';
+    }
+
+    get recordNameFieldClass() {
+        return this.showEnterRecordNameError || this.showInvalidRecordNameError
+            ? 'md-invalid'
+            : '';
+    }
```

Comment:
@TroyceGowdy Using `getFormErrors()` would require these getters to look more like this:
https://github.com/casual-simulation/casualos/blob/e7ff3679133d7b92ef5bbcf6af0a6cc79fd75a8f/src/aux-server/aux-web/shared/vue-components/EnterAccountInfoDialog/EnterAccountInfoDialog.ts#L73-L76